persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -10,6 +10,7 @@ CREATE TABLE Conversation (
     mls_group_id TEXT,
     mls_group_state TEXT AS ConversationEntity.GroupState NOT NULL,
     mls_epoch INTEGER DEFAULT 0 NOT NULL,
+    mls_proposal_timer TEXT,
```

Comment:
since it's timestamp, I think we can set it as long, right?

Reply:
We could but all dates/timestamps are currently stored as TEXT. I'd prefer to do a refactoring PR where we change timestamps to be represented by a long. Someone said has been discussed during the collective.